data/presets/presets/indoor/escalator.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icon":"maki-entrance",
+    "fields": [
+        "level",
+        "name",
+        "conveying"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "tags": {
+        "indoor": "room",
+        "stairs": "yes",
+        "conveying": "yes"
+    },
+    "terms": [
+        "escalator"
+    ],
+    "matchScore": 0.8,
+    "name": "Indoor escalator"
```

Comment:
Hmm a room dedicated to escalators doesn't seem particularly common. Can we just skip this preset and use the `conveying` field for the stairwell preset instead?

Reply:
@quincylvania, I followed the Simple Indoor Tagging scheme as documented as closely as possible. I agree with some of your comments related to the exact wording. However, I'm not sure if we want to deviate from the official recommendation. There have been a few iterations on the scheme and it seems to have stabilised around this.

Link to the details on the connections between levels:
https://wiki.openstreetmap.org/wiki/Simple_Indoor_Tagging#Modelling_connections_between_different_levels

Reply:
I think the tags are okay but adding a preset for it seems potentially confusing (indoor escalator room?) and not super useful.

Reply:
> However, I don't have a strong opinion here so if you would like to merge both I'm fine with that.

Yeah let's just add the stairwell for now and use a field to specify if they stairs are actually escalators. Maybe later I could be convinced otherwise 😆